### ISSUE TITLE
WCAG2AA 1.4.3 G18 Returning inconsistent results

### ISSUE THREAD

**Issue by amrocha** [pa11y/pa11y#422]
I've been trying to add pa11y to a private repo over at Shopify. I'm running into a problem where pa11y is triggering false positives on this rule sometimes, but only in CI. I have not been able to reproduce this locally.

I am passing an instance of browser to pa11y.

## Expected behaviour
Results are consistent

## Actual behaviour
Results are inconsistent, and there's false positives:
 - Most of the time tests pass
 - 10% of builds seem to fail due to this rule
 - Checking the contrast manually shows this rule should not be triggered

## Steps to reproduce
Unfortunately I wasn't able to reproduce this locally. This is happening in CircleCI, using the `circleci/node:8.11.2-browsers` docker image.

## Environment:
Pa11y:      5.0.4
Node.js:    8.11.2
npm:        5.6.0
OS:         4.4.0-133-generic (linux)

**Comment by amrocha** [comment]
>Is it possible that Pa11y is testing the page before its fully loaded?
I thought it might be that at first, but wasn't able to reproduce it locally at all. I'll try again with a wait in CI and see if it keeps happening.

I'l look into screen capture as well.

**Comment by amrocha** [comment]
We haven't run into this since so I'm closing this issue

**Comment by nantunes** [comment]
I'm facing the same issue.

Pa11y: 5.3.0
Node.js: 12.17.0
npm: 6.14.4
Virtualization: kvm
Operating System: RHEV 7.7 server
Kernel: Linux 3.10.0-1062.18.1.el7.x86_64
